nfc/include/nfc_platform.h

```diff
@@ -57,9 +57,9 @@ typedef void (* nfc_lib_cb_resolve_t)(const void * p_ctx, const uint8_t * p_data
  * the NFC library. The function should save the nfc_lib_cb_call function pointer and set
  * the interrupt priority for NFC.
  *
- * @param[in]      nfc_lib_cb_resolve Pointer to the callback resolution function.
- * @param[in, out] p_irq_priority     Pointer to the interrupt priority configuration. You must
- *                                    set its value inside this function.
+ * @param[in]      nfc_lib_cb_call Pointer to the callback resolution function.
```

Comment:
The parameter name doesn't match with the function prototype. Maybe this file is out of sync with the one in the NFC source repository?

Reply:
Updated to nrf-library repository.